### ISSUE TITLE
See that BSD4.x code still works

### ISSUE THREAD

**Issue by ams** [LM-3/chaos#64]
It would be very nice if the old code for BSD4.1/etc still worked on VAX.  This might require some careful pondering on how to use old C and not mix with new C.  I'll leave this for @larsbrinkhoff . :-)

**Comment by larsbrinkhoff** [comment]
Oh, you mean this new code.

**Comment by ams** [comment]
Precisely.  It isn’t new code, but rather “new”.  Mostly shuffled and made to work with C99/C11...

**Comment by ams** [comment]
And we are almost back to the original layout, restored all files that are currently used to what is found in the original tree (K&R declarations even!).  `chsocket` might not work on BSD 4.2, but the `chncp` and `chunix` should at least be easily fixable now.

Much of `cmd` is not populated, and the old interfaces for various hardware is also missing.
